react/features/video-menu/components/web/LocalVideoMenuTriggerButton.js

```diff
@@ -81,6 +88,11 @@ class LocalVideoMenuTriggerButton extends Component<Props> {
      */
     popoverRef: Object;
 
+    /**
+     * Preserve the intial toolbox state.
+     */
+    toolboxEnabled: boolean;
```

Comment:
Could you please rename this property to something like `initialToolboxState`?  It's a bit confusing to have `toolboxEnabled` and `_props.toolboxEnabled`.